pulsar-broker/src/main/java/org/apache/pulsar/broker/web/PulsarWebResource.java

```diff
@@ -992,6 +1004,10 @@ protected boolean isLeaderBroker() {
     }
 
     protected static boolean isLeaderBroker(PulsarService pulsar) {
+        // For extensible load manager, it doesn't have leader election service on pulsar broker.
+        if (pulsar.getLeaderElectionService() == null) {
```

Comment:
Can we not pass this authoritative flag when using the LM extension?

We do have the leader broker in BSC, but here we always return false from the LM extension, which is counter-intuitive.

Reply:
Or plz check if it is the LM extension before and let the authoritative flag false instead of calling isLeader().

Reply:
Updated.